wdl-lint/tests/lints/import-whitespace/source.wdl

```diff
@@ -0,0 +1,21 @@
+## This is a test of import whitespace.
+
+version 1.1
+
+import "foo.wdl"    # OK
+import "bar.wdl"    # OK
+
+import "baz.wdl"    # BAD
+
+import "huh.wdl"    # BAD
+# but a comment makes it OK
+import "vom.wdl"   # OK
+
```

Comment:
I just noticed that this blank line isn't getting a diagnostic and I think that's because of the comment.

I think we may need to change `last_whitespace` to instead look back through _all_ whitespace tokens until we hit the previous import node and count across all whitespace tokens encountered.

Also, now that I think about it, we probably only want to report this diagnostic if there's exactly two lines; any more than that and we'll be triggering the more general `Whitespace` that warns on more than one blank line in a row.

Reply:
Good eyes! I think I addressed all this in what I'm about to commit.